data/covering_grammar/lib/error_analysis.py

```diff
@@ -18,12 +18,30 @@
 
 
 import argparse
+import csv
+import datetime
+import os
 
+import errno
 import prettytable  # type: ignore
 import pynini
 from pynini.lib import rewrite
 
 
+def create_new_log_dir() -> str:
+    error_log_dir = os.path.join(os.getcwd(), "logs")
+    try:
```

Comment:
I think you can just do `exist_ok=True` as an option to `os.makedirs`. I can't think of any reason that would fail, so you can then get rid of the try/except and `errno`.